packages/storage-documents/src/page-scan-run-result.ts

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { StorageDocument } from '.';
+
+export declare type ReportFormat = 'sarif';
+export declare type ScanState = 'unknown' | 'pass' | 'fail';
+export declare type RunState = 'unknown' | 'accepted' | 'queued' | 'running' | 'completed' | 'failed';
+
+/**
+ * The web page scan run result document.
```

Comment:
can you scope all the new interfaces under WebApiFlowDocuments or a new package to make it eaiser to understand where these are used?